app/models/tweet.rb

```diff
@@ -53,13 +46,27 @@ def period_params(period)
       end
     end
 
-    # クエリ指定
-    def fetch_request_query(query_params)
-      "{
-        \"query\":\"from:#{query_params[:login_user]}\",
-        \"fromDate\":\"#{query_params[:date_from]}\",
-        \"toDate\":\"#{query_params[:date_to]}\"
-      }"
+    def sent_query(query_params)
+      data = {
+        "query": "from:#{query_params[:login_user]}".to_s,
+        "fromDate": query_params[:date_from].to_s,
+        "toDate": query_params[:date_to].to_s
+      }
+      data.merge!({ "next": query_params[:next].to_s }) unless query_params[:next].nil?
```

Comment:
```suggestion
      data.merge!({ "next": query_params[:next].to_s }) if query_params[:next]
```
あえてunlessを使う必要ないと思います。